src/mainTemplate.json

```diff
@@ -249,17 +249,17 @@
       }
     },
     "vmDataDiskCount": {
-      "type": "string",
-      "defaultValue": "40",
+      "type": "int",
```

Comment:
We can also revert this https://github.com/elastic/azure-marketplace/pull/84/commits/c7fa33b401e1b2cbb25231cff00abe1d0a869e71 now that this is an int again.